pyrokinetics/examples/example.py

```diff
@@ -2,14 +2,12 @@
 
 gs2input = template_dir / "input.gs2"
 eqfile = template_dir / "test.geqdsk"
-pyro = Pyro(eq_file=eqfile, eq_type="GEQDSK", local_geometry="Miller", gk_type="GS2")
+pyro = Pyro(gk_code="GS2", gk_file=gs2input)
```

Comment:
`eqfile` is now not used?

Reply:
It wasn't really being used before. A lot of the example needed updating